### ISSUE TITLE
Driver.version unknown when running protractor chrome from cmd

### ISSUE THREAD

**Issue by dudumanbogdan** [angular/protractor#3373]
Hi,

I am struggling with this error for some time. Using the same project example on another machine, but windows 7, the chrome is launching.

Can someone help me to fix this?

Here is the full logs:
```
d:\Bogdan\OneDrive\Documents\SimpleProtractorExmple2>webdriver-manager start
[13:10:15] I/start - java -jar C:\Users\duduman\AppData\Roaming\npm\node_modules\protractor\node_modules\webdriver-manager\selenium\selenium-server-standalone-2.53.1.jar -Dwebdriver.chrome.driver=C:\Users\duduman\AppData\Roaming\npm\node_modules\protractor\node_modules\webdriver-manager\selenium\chromedriver_2.22.exe -Dwebdriver.edge.driver=C:\Program Files (x86)\Microsoft Web Driver\MicrosoftWebDriver.exe
[13:10:15] I/start - seleniumProcess.pid: 11240
13:10:16.824 INFO - Launching a standalone Selenium Server
Setting system property webdriver.chrome.driver to C:\Users\duduman\AppData\Roaming\npm\node_modules\protractor\node_modules\webdriver-manager\selenium\chromedriver_2.22.exe
Setting system property webdriver.edge.driver to C:\Program Files (x86)\Microsoft Web Driver\MicrosoftWebDriver.exe
13:10:17.047 INFO - Java: Oracle Corporation 25.91-b15
13:10:17.048 INFO - OS: Windows 10 10.0 x86
13:10:17.093 INFO - v2.53.1, with Core v2.53.1. Built from revision a36b8b1
13:10:17.230 INFO - Driver class not found: com.opera.core.systems.OperaDriver
13:10:17.232 INFO - Driver provider com.opera.core.systems.OperaDriver is not registered
13:10:17.241 INFO - Driver provider org.openqa.selenium.safari.SafariDriver registration is skipped:
registration capabilities Capabilities [{browserName=safari, version=, platform=MAC}] does not match the current platform WIN10
13:10:17.245 INFO - Driver class not found: org.openqa.selenium.htmlunit.HtmlUnitDriver
13:10:17.247 INFO - Driver provider org.openqa.selenium.htmlunit.HtmlUnitDriver is not registered
13:10:17.401 INFO - RemoteWebDriver instances should connect to: http://127.0.0.1:4444/wd/hub
13:10:17.403 INFO - Selenium Server is up and running
13:10:45.254 INFO - Executing: [new session: Capabilities [{count=1, browserName=chrome, chromeOptions={args=[user-data-dir=C:/Users/rai.AD/AppData/Local/Google/Chrome/User Data/Default]}}]])
13:10:45.281 INFO - Creating a new session for Capabilities [{count=1, browserName=chrome, chromeOptions={args=[user-data-dir=C:/Users/rai.AD/AppData/Local/Google/Chrome/User Data/Default]}}]
Starting ChromeDriver 2.22.397933 (1cab651507b88dec79b2b2a22d1943c01833cc1b) on port 17339
Only local connections are allowed.
13:10:47.168 WARN - Exception thrown
java.util.concurrent.ExecutionException: org.openqa.selenium.WebDriverException: java.lang.reflect.InvocationTargetException
Build info: version: '2.53.1', revision: 'a36b8b1', time: '2016-06-30 17:37:03'
System info: host: 'DESKTOP-M05TSGQ', ip: '192.168.170.50', os.name: 'Windows 10', os.arch: 'x86', os.version: '10.0', java.version: '1.8.0_91'
Driver info: driver.version: unknown
        at java.util.concurrent.FutureTask.report(Unknown Source)
        at java.util.concurrent.FutureTask.get(Unknown Source)
        at org.openqa.selenium.remote.server.DefaultSession.execute(DefaultSession.java:183)
        at org.openqa.selenium.remote.server.DefaultSession.<init>(DefaultSession.java:119)
        at org.openqa.selenium.remote.server.DefaultSession.createSession(DefaultSession.java:95)
        at org.openqa.selenium.remote.server.DefaultDriverSessions.newSession(DefaultDriverSessions.java:124)
        at org.openqa.selenium.remote.server.handler.NewSession.handle(NewSession.java:59)
        at org.openqa.selenium.remote.server.handler.NewSession.handle(NewSession.java:1)
        at org.openqa.selenium.remote.server.rest.ResultConfig.handle(ResultConfig.java:111)
        at org.openqa.selenium.remote.server.JsonHttpCommandHandler.handleRequest(JsonHttpCommandHandler.java:79)
        at org.openqa.selenium.remote.server.DriverServlet.handleRequest(DriverServlet.java:204)
        at org.openqa.selenium.remote.server.DriverServlet.doPost(DriverServlet.java:166)
        at javax.servlet.http.HttpServlet.service(HttpServlet.java:707)
        at org.openqa.selenium.remote.server.DriverServlet.service(DriverServlet.java:132)
        at javax.servlet.http.HttpServlet.service(HttpServlet.java:790)
        at org.openqa.jetty.jetty.servlet.ServletHolder.handle(ServletHolder.java:428)
        at org.openqa.jetty.jetty.servlet.ServletHandler.dispatch(ServletHandler.java:680)
        at org.openqa.jetty.jetty.servlet.ServletHandler.handle(ServletHandler.java:571)
        at org.openqa.jetty.http.HttpContext.handle(HttpContext.java:1526)
        at org.openqa.jetty.http.HttpContext.handle(HttpContext.java:1479)
        at org.openqa.jetty.http.HttpServer.service(HttpServer.java:920)
        at org.openqa.jetty.http.HttpConnection.service(HttpConnection.java:820)
        at org.openqa.jetty.http.HttpConnection.handleNext(HttpConnection.java:986)
        at org.openqa.jetty.http.HttpConnection.handle(HttpConnection.java:837)
        at org.openqa.jetty.http.SocketListener.handleConnection(SocketListener.java:243)
        at org.openqa.jetty.util.ThreadedServer.handle(ThreadedServer.java:358)
        at org.openqa.jetty.util.ThreadPool$PoolThread.run(ThreadPool.java:537)
Caused by: org.openqa.selenium.WebDriverException: java.lang.reflect.InvocationTargetException
Build info: version: '2.53.1', revision: 'a36b8b1', time: '2016-06-30 17:37:03'
System info: host: 'DESKTOP-M05TSGQ', ip: '192.168.170.50', os.name: 'Windows 10', os.arch: 'x86', os.version: '10.0', java.version: '1.8.0_91'
Driver info: driver.version: unknown
        at org.openqa.selenium.remote.server.DefaultDriverProvider.callConstructor(DefaultDriverProvider.java:113)
        at org.openqa.selenium.remote.server.DefaultDriverProvider.newInstance(DefaultDriverProvider.java:97)
        at org.openqa.selenium.remote.server.DefaultDriverFactory.newInstance(DefaultDriverFactory.java:60)
        at org.openqa.selenium.remote.server.DefaultSession$BrowserCreator.call(DefaultSession.java:222)
        at org.openqa.selenium.remote.server.DefaultSession$BrowserCreator.call(DefaultSession.java:1)
        at java.util.concurrent.FutureTask.run(Unknown Source)
        at org.openqa.selenium.remote.server.DefaultSession$1.run(DefaultSession.java:176)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
        at java.lang.Thread.run(Unknown Source)
Caused by: java.lang.reflect.InvocationTargetException
        at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
        at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
        at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
        at java.lang.reflect.Constructor.newInstance(Unknown Source)
        at org.openqa.selenium.remote.server.DefaultDriverProvider.callConstructor(DefaultDriverProvider.java:103)
        ... 9 more
Caused by: org.openqa.selenium.WebDriverException: unknown error: cannot create default profile directory
  (Driver info: chromedriver=2.22.397933 (1cab651507b88dec79b2b2a22d1943c01833cc1b),platform=Windows NT 10.0.10586 x86_64) (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 723 milliseconds
Build info: version: '2.53.1', revision: 'a36b8b1', time: '2016-06-30 17:37:03'
System info: host: 'DESKTOP-M05TSGQ', ip: '192.168.170.50', os.name: 'Windows 10', os.arch: 'x86', os.version: '10.0', java.version: '1.8.0_91'
Driver info: org.openqa.selenium.chrome.ChromeDriver
        at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
        at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
        at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
        at java.lang.reflect.Constructor.newInstance(Unknown Source)
        at org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:206)
        at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:158)
        at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:678)
        at org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:249)
        at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:131)
        at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:144)
        at org.openqa.selenium.chrome.ChromeDriver.<init>(ChromeDriver.java:170)
        at org.openqa.selenium.chrome.ChromeDriver.<init>(ChromeDriver.java:138)
        ... 14 more
13:10:47.321 WARN - Exception: unknown error: cannot create default profile directory
  (Driver info: chromedriver=2.22.397933 (1cab651507b88dec79b2b2a22d1943c01833cc1b),platform=Windows NT 10.0.10586 x86_64) (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 723 milliseconds
Build info: version: '2.53.1', revision: 'a36b8b1', time: '2016-06-30 17:37:03'
System info: host: 'DESKTOP-M05TSGQ', ip: '192.168.170.50', os.name: 'Windows 10', os.arch: 'x86', os.version: '10.0', java.version: '1.8.0_91'
Driver info: org.openqa.selenium.chrome.ChromeDriver
```

And here is the protractor.conf.js:
```
exports.config = {
    seleniumAddress: 'http://localhost:4444/wd/hub',
    baseURL: 'http://google.com,
    rootElement: 'cm-app',
    specs: ['*.spec.js'],

    capabilities: {
        'browserName': 'chrome'
        },

    onPrepare: function () {
              browser.driver.manage().window().maximize();//setSize(1920, 1080);
          },
    getPageTimeout: 60000,
    allScriptsTimeout: 60000,
    useAllAngular2AppRoots: true,

    jasmineNodeOpts: {
        isVerbose: true,
        showColors: true,
        includeStackTrace: true,
        defaultTimeoutInterval: 9999000
    }
};
```

**Comment by dudumanbogdan** [comment]
Also I will post the versions I am using:
>npm -v
3.10.5

>node -v
v5.10.1

>webdriver-manager list
[13:18:46] I/status - selenium standalone version available: 2.53.1 [default]
[13:18:46] I/status - chromedriver version available: 2.22 [default]
[13:18:46] I/status - IEDriverServer is not present
[13:18:46] I/status - android-sdk is not present
[13:18:46] I/status - appium is not present

**Comment by cnishina** [comment]
This appears to be a support question because it appears to be a set up problem with the local machine. I am unable to reproduce this on Windows 10 with node version 4, chrome 51, protractor 4. This question might be better suited for [StackOverflow](http://stackoverflow.com/questions/tagged/protractor) or [Gitter](https://gitter.im/angular/protractor).

There are a couple of things you could check:

* chrome is at the latest version
* run `webdriver-manager clean` then `webdriver-manager update`. This will clear the binaries and download them again.

In a totally unrelated issue, it appears that webdriver-manager automatically is setting the edge location whether or not the edge driver has been installed. Not a high priority, but this should eventually be fixed. Tagging this issue for logs.
